vars/perfRegressionParallelPipeline.groovy

```diff
@@ -64,13 +64,15 @@ def call(Map pipelineParams) {
                         } else {
                             sub_tests = [pipelineParams.test_name]
                         }
-                        for (sub_test in sub_tests) {
+                        for (t in sub_tests) {
                             def perf_test
+                            def sub_test = t
```

Comment:
why this trick is necessary?

Reply:
cause of parallel jobs in the jenkinsfile